packages/wonder-blocks-testing/src/fixtures/adapters/storybook.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import * as React from "react";
+import {action} from "@storybook/addon-actions";
+import {Adapter} from "./adapter.js";
+
+import type {
+    AdapterGroupOptions,
+    AdapterFixtureOptions,
+    AdapterFactory,
+} from "../types.js";
+
+type StoryContext = {|
+    args: $ReadOnly<any>,
+    argTypes: $ReadOnly<any>,
+    globals: $ReadOnlyArray<any>,
+    hooks: $ReadOnlyArray<any>,
+    parameters: $ReadOnly<any>,
+    viewMode: mixed,
+|};
```

Comment:
TIL that StoryBook has its own context object that we can use.

Reply:
Yeah, I learned while I was reading up on StoryBook's API to try and flesh out this framework adapter a bit.